src/Network/Network/help/Reset-HubRouter.md

```diff
@@ -0,0 +1,225 @@
+---
```

Comment:
can you check why this generated help doesn't have Az prefix in cmdlet name:
Reset-HubRouter => ResetAzHubRouter

Reply:
Hi @nagula-ritvika ,

the name is still incorrect in this file, please regenerate a proper help markdown